lib/grammers-client/src/types/input_media.rs

```diff
@@ -0,0 +1,283 @@
+// Copyright 2020 - developers of the `grammers` project.
+//
+// Licensed under the Apache License, Version 2.0 <LICENSE-APACHE or
+// https://www.apache.org/licenses/LICENSE-2.0> or the MIT license
+// <LICENSE-MIT or https://opensource.org/licenses/MIT>, at your
+// option. This file may not be copied, modified, or distributed
+// except according to those terms.
+use super::attributes::Attribute;
+use crate::types::{Media, Uploaded};
+use grammers_tl_types as tl;
+
+/// Construct and send albums.
+#[derive(Default)]
+pub struct InputMedia {
+    pub(crate) entities: Vec<tl::enums::MessageEntity>,
+    pub(crate) reply_to: Option<i32>,
+    pub(crate) caption: String,
+    pub(crate) media: Option<tl::enums::InputMedia>,
+    media_ttl: Option<i32>,
+    mime_type: Option<String>,
+}
+
+impl InputMedia {
+    /// The formatting entities within the caption (such as bold, italics, etc.).
+    pub fn fmt_entities(mut self, entities: Vec<tl::enums::MessageEntity>) -> Self {
+        self.entities = entities;
+        self
+    }
+
+    /// The album identifier to which this album should reply to, if any.
+    ///
+    /// Otherwise, this album will not be a reply to any other.
+    pub fn reply_to(mut self, reply_to: Option<i32>) -> Self {
+        self.reply_to = reply_to;
+        self
+    }
```

Comment:
Might want to mention that *only* the reply from the *first* media is used.

Reply:
sure